generators/pkg/writer/write.go

```diff
@@ -0,0 +1,20 @@
+// Package writer provides a method to write the generated task
+// to the yaml document.
+package writer
+
+import (
+	"log"
+
+	v1beta1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	"sigs.k8s.io/yaml"
+)
+
+// WriteToDisk writes the generated task
+// to the []byte
+func WriteToDisk(task v1beta1.Task) ([]byte, error) {
+	data, err := yaml.Marshal(&task)
+	if err != nil {
+		log.Fatal(err)
```

Comment:
You should avoid calling log.Fatal in library funcs, since `log.Fatal` will cause the program to immediately exit, which severely limits the ability for the func to be reused. In general you should return the error and let the caller decide how to handle it.